solutions/5.js

```diff
@@ -47,7 +47,31 @@ const solution2 = (arr) => {
   }
   return arr;
 };
+
+// Betty Chow
+// sort numbers or strings in an array in ascending order
+const solution1 = (arr) => {
+    let newArr = [];
+    while( arr.length > 0) {
+      let smallestNum = arr[0];
+      for(let i=0; i < arr.length; i++) {
+        if (arr[i] < smallestNum) {
+          smallestNum = arr[i];
+        }
+      }
+      let x = arr.indexOf(smallestNum);
+      newArr.push(arr.splice(x , 1));
```

Comment:
since you are always only removing one element, you can directly push the first element `newArr.push(arr.splice(x , 1)[0]);`

Then, you won't have to use a new var called `finalArr` or `concat.apply` function in line 65.